modules/dynamic-plugins/proc-package-third-party-dynamic-plugin.adoc

```diff
@@ -0,0 +1,158 @@
+[id="proc-package-third-party-dynamic-plugin_{context}"]
+= Packaging and publishing third-party plugins as dynamic plugins
+
+To package a third-party plugin as a dynamic plugin, you need to access to its source code and must export a plugin using the `@janus-idp/cli`. For more information, see xref:proc-export-third-party-plugins-rhdh_{context}[]. 
+
+After exporting a third-party plugin, you can package the derived package into one of the following supported formats:
+
+* Open Container Initiative (OCI) image (recommended)
+* TGZ file
+* JavaScript package
+
```

Comment:
[IMPORTANT]
====
Red Hat recommends packaging dynamic plugins into Open Container Initiative (OCI) image 
====

Reply:
(Recommended) is added along with the Open Container Initiative (OCI) image in the first bullet.